internal/characters/sayu/skill.go

```diff
@@ -76,7 +76,12 @@ func (c *char) skillPress(p map[string]int) action.ActionInfo {
 		Mult:       skillPress[c.TalentLvlSkill()],
 	}
 	snap := c.Snapshot(&ai)
-	c.Core.QueueAttackWithSnap(ai, snap, combat.NewCircleHit(c.Core.Combat.Player(), 3), skillPressDoTHitmark)
+	c.Core.QueueAttackWithSnap(
+		ai,
+		snap,
+		combat.NewCircleHitOnTarget(c.Core.Combat.Player(), nil, 3),
+		skillPressDoTHitmark,
+	)
```

Comment:
This is an approximation, because the tick dmg is pure single target and it happens on collision between Sayu and the enemy. I'll change it to be pure ST on PrimaryTarget because this shouldn't actually be AoE.

Reply:
Tracked in #1177.